tests/pinkopy/test_helper.py

```diff
@@ -0,0 +1,52 @@
+import requests_mock
+import uuid
+
+
+def get_uuid():
+    return str(uuid.uuid4())
+
+
+def mock_session(base_session, service='http://' + get_uuid(),
+                 user=get_uuid(), pw=get_uuid(), token=get_uuid(),
+                 content_type='application/json', clients=get_uuid()
+                 ):
```

Comment:
I'm curious, what do we gain here over:
```python
def mock_session(base_session, service='http://example.com',
                 user='user', pw='pw', token='token',
                 content_type='application/json', clients=['client1' , 'client2'])
```
Of course, what you did is perfectly suitable, since the performance of creating unique strings is not relevant when testing. I'm just interested.